Updated AE Extension installation instructions (#217)
ynput/ayon-documentation: website/docs/addon_aftereffects_artist.md

-   [Publish](artist_tools_publisher)
-   [Manage](artist_tools_inventory)

## AE Extension installation

:::tip
Once your admin has activated `After Effects` in the production bundle, you need to run the AYON launcher at least once. This allows the launcher to download the addon to the correct location on your computer. 
After that, you can continue with the installation instructions for the After Effects extension.
:::

<Tabs
groupId="platforms"
defaultValue="win"
values={[
{label: 'Windows', value: 'win'},
{label: 'Linux', value: 'linux'},
{label: 'Mac', value: 'mac'},
]}>

<TabItem value="win">

1. Use your favorite app to install extensions. we recommend using [Anastasyi's Extension Manager](https://install.anastasiy.com/).

> 2. Locate and install the `extension.zxp` file. You can find it in the `AppData` folder, typically at `c:\Users\YOUR_USER\AppData\Local\Ynput\AYON\addons\aftereffects_X.X.X\ayon_aftereffects\api`. Alternatively, you can quickly navigate there using the environment variable shortcut: `%LOCALAPPDATA%\Ynput\AYON\addons\aftereffects_X.X.X\ayon_aftereffects\api`.

Reviewer: ```suggestion
2. Locate and install the `extension.zxp` file. You can find it in the `AppData` folder, typically at `c:\Users\YOUR_USER\AppData\Local\Ynput\AYON\addons\aftereffects_X.X.X\ayon_aftereffects\api`. 

:::tip
You can quickly navigate to the local app data using the environment variable shortcut: `%LOCALAPPDATA%`. For example, using `%LOCALAPPDATA%\Ynput\AYON\addons` as path to see your installed addons.
:::
```

Reviewer 2: Thank you https://github.com/ynput/ayon-documentation/pull/217/commits/5e17173db0cadc8d453509a25a7f6d819216e2a1

</TabItem>

<TabItem value="linux">

1. Use your favorite app to install extensions.
2. Locate and install the `extension.zxp` file. You can find it in the user data folder, typically at `~/.local/share/Ynput/AYON/addons/aftereffects_X.X.X/ayon_aftereffects/api`.

</TabItem>

<TabItem value="mac">

1. Use your favorite app to install extensions. we recommend using [Anastasyi's Extension Manager](https://install.anastasiy.com/).

> 2. Locate and install the `extension.zxp` file. You can find it in the `Application Support` folder, typically at`~/Library/Application Support/Ynput/AYON/addons/aftereffects_X.X.X/ayon_aftereffects/api`.

Reviewer: ```suggestion
2. Locate and install the `extension.zxp` file. You can find it in the `Application Support` folder, typically at `~/Library/Application Support/Ynput/AYON/addons/aftereffects_X.X.X/ayon_aftereffects/api`.
```

Reviewer 2: Good catch. Thank you.
https://github.com/ynput/ayon-documentation/pull/217/commits/5e17173db0cadc8d453509a25a7f6d819216e2a1

-   [Publish](artist_tools_publisher)
-   [Manage](artist_tools_inventory)

## AE Extension installation

:::info
Once your admin has activated `After Effects` in the production bundle, you need to run the AYON launcher at least once. This allows the launcher to download the addon to the correct location on your computer. 
After that, you can continue with the installation instructions for the After Effects extension.
:::

<Tabs
groupId="platforms"
defaultValue="win"
values={[
{label: 'Windows', value: 'win'},
{label: 'Linux', value: 'linux'},
{label: 'Mac', value: 'mac'},
]}>

<TabItem value="win">

> 1. Use your favorite app to install extensions. we recommend using [Anastasyi's Extension Manager](https://install.anastasiy.com/).

Reviewer: ```suggestion
1. Use your favorite app to install extensions. We recommend using [Anastasyi's Extension Manager](https://install.anastasiy.com/).
```

2. Locate and install the `extension.zxp` file. You can find it in the `AppData` folder, typically at `c:\Users\YOUR_USER\AppData\Local\Ynput\AYON\addons\aftereffects_X.X.X\ayon_aftereffects\api`.

:::tip
You can quickly navigate to the local app data using the environment variable shortcut: `%LOCALAPPDATA%`. For example, using `%LOCALAPPDATA%\Ynput\AYON\addons` as path to see your installed addons.
:::

</TabItem>

<TabItem value="linux">

> 1. Use your favorite app to install extensions.

Reviewer: Interesting - I can't find anything about installing adobe extensions on Linux. Are adobe products even supported on Linux?

Reviewer 2: Quoted: 
```
= Can I run Photoshop on Ubuntu?
Adobe Photoshop is not natively available for Ubuntu. 
However, it can be installed on Ubuntu using the Wine compatibility layer. 
What is Wine? Wine is a compatibility layer that allows users to run Windows applications 
on Linux, macOS, and other Unix-like operating systems.Apr 13, 2022
```

Reviewer: If so, we may actually not 'hint' at using their favorite app to install it and just say it's e.g. not officially supported on Linux? Or?

Reviewer 2: @m-u-r-p-h-y  @kalisp what do you think we should say about it ?

Author: If they are interested how to install AYON integration for AE on LInux, they must figured out somehow how to install it, so   this documentation should be still applicable.

2. Locate and install the `extension.zxp` file. You can find it in the user data folder, typically at `~/.local/share/Ynput/AYON/addons/aftereffects_X.X.X/ayon_aftereffects/api`.

</TabItem>

<TabItem value="mac">

> 1. Use your favorite app to install extensions. we recommend using [Anastasyi's Extension Manager](https://install.anastasiy.com/).

Reviewer: ```suggestion
1. Use your favorite app to install extensions. We recommend using [Anastasyi's Extension Manager](https://install.anastasiy.com/).
```

## Setup

To install the extension, download, install [Anastasyi's Extension Manager](https://install.anastasiy.com/). Open Anastasyi's Extension Manager and select AfterEffects in menu. 

Use extension.zxp downloaded with AfterEffects addon.

> Current location will be in artist `AppData`, on Windows it would be something like `c:\Users\YOUR_USER\AppData\Local\Ynput\AYON\addons\aftereffects_X.X.X\ayon_aftereffects\api`.

Reviewer: I wonder if it'd be nice to add a TIP section that you can easily browse to the folder using `%LOCALAPPDATA%`
```suggestion
Current location will be in artist `AppData`, on Windows it would be something like `c:\Users\YOUR_USER\AppData\Local\Ynput\AYON\addons\aftereffects_X.X.X\ayon_aftereffects\api`.
:::tip 
To quickly browse to the local appdata folder on your machine, use `%LOCALAPPDATA%` as path in the Windows explorer.
:::
```

Reviewer: No. Just below it mentions it for Linux and mac?

Reviewer 2: Thank you. added here https://github.com/ynput/ayon-documentation/pull/217/commits/57c13df08208942748e47cfae925082acc9c7e25

On Linux look in `~/.local/share/Ynput/AYON/addons/aftereffects_X.X.X/ayon_aftereffects/api`, on Mac `~/Library/Application Support/Ynput/AYON/addons/aftereffects_X.X.X/ayon_aftereffects/api`.

> Artist must start Tray at least once for Ayon server to populate this location. Extension should be installed before AE is opened through Launcher!

Reviewer: ```suggestion

:::note
You must start tray at least once for the AYON launcher with After Effects enabled in your bundle to populate this location. However, the extension should be installed before After Effects is opened through Launcher.
:::
```

Reviewer 2: Good idea. added here https://github.com/ynput/ayon-documentation/pull/217/commits/57c13df08208942748e47cfae925082acc9c7e25
